kivymd/uix/snackbar.py

```diff
@@ -402,6 +402,10 @@ def wait_interval(interval):
                 Clock.unschedule(wait_interval)
                 self._interval = 0
 
+        for c in Window.parent.children:
+            if issubclass(type(c), Snackbar):
+                return
+
```

Comment:
i would recommend you to use isinstance instead of issubclass.
if the object is not directly a subclass of the snackbar, the code block wouldn't return, this means that the current snackbar will try to show up over the other one.